integrations/flink/documentation.yaml

```diff
@@ -0,0 +1,79 @@
+exporter_type: included
+app_name_short: Flink
+app_name: {{app_name_short}}
+app_site_name: Flink
+app_site_url: https://flink.apache.org/
+exporter_name: the Flink exporter
+exporter_pkg_name: Flink
+exporter_repo_url: https://flink.apache.org/features/2019/03/11/prometheus-monitoring.html
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics when configured with
+  `metrics.reporter.prom.class: org.apache.flink.metrics.prometheus.PrometheusReporter`.
+
+   *  To determine if {{app_name_short}} is exposing metrics, run the following command:
+
+      <pre class="devsite-click-to-copy">
+      kubectl -n <namespace_name> port-forward <flink_pod_name> 9249
+      curl localhost:9249/metrics
+      </pre>
+dashboard_available: true
+multiple_dashboards: false
```

Comment:
multiple_dashboards should be true?

Reply:
Tagging @mtabasko  -- we don't have many docs with `multiple_dashboards = true` -- in this case, the template no longer uses `dashboard_display_name`, right? If so, let's clean it up.